### PR TITLE
Add basic immunisation and booster vaccines to glossary

### LINKED ISSUE #3165: Add new terms reg. vaccination statistics to the Glossary
## Enhancement description
Please add the definitions of the following terms (and their German equivalents listed in brackets) to the [Glossary](https://www.coronawarn.app/en/faq/results/?search=&topic=glossary):
- "People Who Have Received at Least One Vaccine Dose" ("Mindestens eimal geimpfte Personen")
- "People Who Have Basic Immunization" ("Grundimmunisiert")
- "People Who Have Received a Booster Vaccination" ("Personen mit Auffrischimpfung")

## Problem and motivation
The app includes the above-mentioned terms without explaining them. It would be good to at least have a definition of the terms in the CWA website glossary. 

## Is this something you're interested in working on
No, this should be done by the team.

## Additional information
This is a follow-up to https://github.com/corona-warn-app/cwa-documentation/issues/922

---
Internal Tracking ID: [EXPOSUREAPP-14220](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14220)

### PR DESCRIPTION
**Pull request by brianebeling**
Fixes #3165

Texts are taken from: https://impfdashboard.de/en/

en
image

de
image
image

---
Internal Tracking ID: [EXPOSUREAPP-14220](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14220)

### CODE REVIEW

**Comment by GisoSchroederSAP** [comment]
Guys, this needs to be carefully reviewed to avoid we put something in the glossary, that is not in sync with the official definition of the legislative body. All input is welcome, but please don't merge before we have a "go" from the experts.
Thx in advance!

**Comment by GisoSchroederSAP** [comment]
The "Basic Immunization" (correct term: _Primary_ Immunization) needs to be adjusted please, because the current description does not cover the acceptable cases. Please use this explanation instead: 

> Als grundimmunisiert zählen alle Menschen, die die initiale Impfserie durchlaufen haben. Bei SARS-CoV-2 umfasst diese im Allgemeinen zwei Impfdosen, bei nachweislich Genesenen aber nur eine einzelne Impfdosis. Auch Personen, die bereits weitere Impfdosen erhalten haben, werden hier mitgezählt. Eine vollständige Grundimmunisierung ist nicht gleichbedeutend mit einem vollständigen Impfschutz.

image
(Source: Fachwörterbuch Infektionsschutz und Infektionsepidemiologie)

As a result, also the term "Auffrischimpfung" can be simplified:

> Als Geimpfte mit einer Auffrischimpfung zählen alle Menschen, die grundimmunisiert [hier ggf. Querverlinkung?] sind und danach mindestens eine weitere Impfdosis erhalten haben.

Thank you.

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

If the term "Basic Immunization" is incorrect then it would need to be changed in the app. See https://www.coronawarn.app/en/screenshots/#android_stats.

**Comment by GisoSchroederSAP** [comment]
For consistency, let's stay with "basic communication", otherwise we generate new irritation and (in worst case) need to go through all screens with UI and translation again. ItÄs not worth this effort. I just wanted to mention the source where the definition for "Grundimmunisierung" comes from.

File: src/data/faq.json [hunk 1]
```diff
@@ -3237,11 +3237,21 @@
                     "anchor": "backup",
                     "description": "Backups refer to versions of a project at a certain point in time in the past, which can later be used to restore a project to its previous state."
                 },
+                {
+                    "term": "Basic Immunisation",
+                    "anchor": "basic_immunisation",
```
Comment (MikeMcC399) [inline src/data/faq.json:3242]: Suggest to use the exact spelling used in the app i.e. Immunization, not Immunisation, also for the anchor. (The app normally uses US not UK spelling.)

File: src/data/faq_de.json [hunk 2]
```diff
@@ -3445,6 +3450,11 @@
                     "anchor": "github",
                     "description": "GitHub ist eine Plattform, die es vielen verschiedenen Personen ermöglicht, gleichzeitig an einem Projekt zu arbeiten. Sie kann die Beiträge aller Personen zusammenführen und dokumentiert jegliche Änderungen."
                 },
+                {
+                    "term": "Grundimmunisierung",
+                    "anchor": "basic_immunisation",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:3455]: Suggest:
"anchor": "basic_immunisation" => "anchor": "basic_immunization"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please use US immunization spelling (with "z") as used in the app.
See for instance screenshots:

- https://www.coronawarn.app/en/screenshots/#android_stats and
- https://www.coronawarn.app/en/screenshots/#ios_stats

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> For consistency, let's stay with "basic communication", otherwise we generate new irritation and (in worst case) need to go through all screens with UI and translation again. ItÄs not worth this effort. I just wanted to mention the source where the definition for "Grundimmunisierung" comes from.

I guess you meant to write "basic immunization" not "basic communication"!

I found that the UK National Health Service (NHS) uses the term "primary immunization" in agreement with your dictionary, however I agree that it is probably too much effort to request and get approval to use the term "primary immunization" everywhere instead. 

"basic immunization" is not only used on the statistics tiles in the app it is also used in the booster notification rule [BNR-DE-0400](https://timokoenig.github.io/eu-dcc-rules/?ruleset=de-bn-rules&country=DE&rule=BNR-DE-0400) "The German Standing Committee on Vaccinations (STIKO) recommends a second booster vaccination for everyone age 60 and older who had their first booster vaccination more than 6 months ago. This recommendation is also valid for persons who were diagnosed with coronavirus after their **basic immunization**, if the infection occurred more than 6 months ago."

**Comment by GisoSchroederSAP** [comment]
You're right, it should have been "basic immunization" (instead of _communication_ ). Let's see if we can gradually introduce the "primary immunization" term by just using it in the communication as often as possible.

**Comment by brianebeling** [comment]
> In addition the Federal Ministry of Health uses "basic immunisation" (UK spelling!) on their English-language website https://impfdashboard.de/en/:
> 
> image

That is also what I oriented myself at. That said, I will implement the requested changes and report back. Thanks!

**Comment by brianebeling** [comment]
What do you think about linking to the new glossary items from everywhere?

Like this for example:
image

It improves navigation, but readability or aesthetics may suffer a bit from a lot of links.

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq.json:3250]: This conversation is marked as resolved although there are still discrepancies in the spelling between the header (immunisation) and the text (immunization).

File: src/data/faq.json [hunk 3]
```diff
@@ -3245,11 +3245,21 @@
...
```
[the remark this answers is not in the document]
Comment (brianebeling) [reply src/data/faq.json:3250]: Sorry, they are resolved locally, but I didn't push yet. I will try to not resolve comments, unless pushed, in the future.

[the remark this answers is not in the document]
Comment (MikeMcC399) [reply src/data/faq.json:3250]: > I will try to not resolve comments, unless pushed, in the future.

That would be better, to keep the status and the PR contents in sync.

File: src/data/faq.json [hunk 4]
```diff
@@ -3245,11 +3245,21 @@
                     "anchor": "backup",
                     "description": "Backups refer to versions of a project at a certain point in time in the past, which can later be used to restore a project to its previous state."
                 },
+                {
+                    "term": "Basic Immunization",
+                    "anchor": "basic_immunization",
+                    "description": "All people who have undergone the initial vaccination series are considered to be primed. In the case of SARS-CoV-2, this generally comprises two vaccine doses, but only a single vaccine dose for those who have been proven to have recovered. Individuals who have already received additional doses of vaccination are also counted here. Complete basic immunization is not equivalent to complete vaccine protection."
```
Comment (MikeMcC399) [inline src/data/faq.json:3251]: "to be primed" sounds strange in this context. See https://dictionary.cambridge.org/dictionary/english/prime (verb)

suggest as replacement:
"to have completed basic immunization"

**Comment by MikeMcC399** [comment]
@brianebeling 

This PR is still in draft status, so not inviting reviews at this time, however I wanted to mention that there are multiple malformed links with `<a/>` which should be `</a>`.

Cypress will check these for you and fail, especially the `faq_link_attr` test.

**Comment by MikeMcC399** [comment]
@brianebeling 

> What do you think about linking to the new glossary items from everywhere?

Good idea. This needs to be done also for the English FAQ, not just the German one though.

**Comment by MikeMcC399** [comment]
@brianebeling 

For easier reading please mark the appropriate conversations above as [resolved](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#resolving-conversations).

File: src/data/faq.json [hunk 5]
```diff
@@ -515,7 +515,7 @@
...
-                                    "See also:<ul><li><a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank'>CWA 2.10: Corona-Warn-App ready to inform users about booster vaccinations</a></li><li><a href='../../blog/2022-06-29-cwa-2-24/' target='_blank' >CWA 2.24: All certificates in one PDF document</a></li><li><a href='../../blog/2022-09-28-cwa-2-27/' target='_blank'>CWA 2.27: Project team adapts CWA to new Infection Protection Act</a></li><li>FAQ: <a href='#eu_dcc_export_error' target='_blank'>Error 'Cannot display print version'</a></li></ul>"
+                                    "See also:<ul><li><a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank'>CWA 2.10: Corona-Warn-App ready to inform users about <a href='#glossary_booster_vaccination' target='_blank'>booster vaccination</a>s</a></li><li><a href='../../blog/2022-06-29-cwa-2-24/' target='_blank' >CWA 2.24: All certificates in one PDF document</a></li><li><a href='../../blog/2022-09-28-cwa-2-27/' target='_blank'>CWA 2.27: Project team adapts CWA to new Infection Protection Act</a></li><li>FAQ: <a href='#eu_dcc_export_error' target='_blank'>Error 'Cannot display print version'</a></li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:518]: Put the "s" from "vaccinations" inside the link

File: src/data/faq.json [hunk 6]
```diff
@@ -738,7 +738,7 @@
...
+                                    "<ul> <li>Only certificates relevant to vaccination status can be renewed, such as <a href='#glossary_booster_vaccination' target='_blank'>booster vaccination</a>s or certificates of recovery.</li> <li>The certificate is valid for longer than 28 days.</li> <li>The certificate already expired more than 90 days ago. In this case, reissuing within the app is no longer possible. Please contact a pharmacy to obtain a new certificate.</li> <li>Only the vaccination certificate with the latest date of issue can be reissued.</li> <li>Only certificates issued in Germany can be renewed via the app.</li> <li>The number of reissues is limited to three per certificate.</li> <li>Your Corona-Warn-App has not been updated to version 2.23. Please perform an update.</li> </ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:741]: It looks strange here as well.
Change to `vaccinations</a>`

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
In English I suggest not to split the link when a plural is involved:
i.e. make it `vaccinations</a>`

I marked up several places, but not all of them.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Same comment for `Auffrischimpfung</a>en` 
Put the "en" inside the link.

File: src/data/faq_de.json [hunk 7]
```diff
@@ -515,7 +515,7 @@
...
+                                    "Siehe auch:<ul><li><a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank'>CWA 2.10: Corona-Warn-App ist bereit, schnell über <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a>en zu informieren</a></li><li><a href='../../blog/2022-06-29-cwa-2-24/' target='_blank' >CWA 2.24: Alle Zertifikate in einem PDF-Dokument</a></li><li><a href='../../blog/2022-09-28-cwa-2-27/' target='_blank'>CWA 2.27: Projektteam passt CWA an neues Infektionsschutzgesetz an</a></li><li>FAQ: <a href='#eu_dcc_export_error' target='_blank'>Fehlermeldung 'Druckversion anzeigen nicht möglich'</a></li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq_de.json:518]: This is a link inside a link. I don't think that makes sense!

**Review by GisoSchroederSAP — APPROVED** [review]
Agreed, put the plural form for booster vaccination inside the link to the glossary, even if the glossary entry is the singular. Whoever has a problem to distinguish, I will answer, that the explanation for a single booster event does not differs at all for multiple booster vaccinations.

On the "basic immunization", IMHO there is no plural 's' required, 'cause, each person can only reach the basic immunization once. afterwards it will never be a "basic" but always be an "booster" immunization, right?

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> On the "basic immunization", IMHO there is no plural 's' required, 'cause, each person can only reach the basic immunization once. afterwards it will never be a "basic" but always be an "booster" immunization, right?

There is no occurrence of "immunizations" (in the plural) on the website, so this isn't an issue.

I am puzzled why you approved this PR. Normally approval is only given when the contents are 100% correct and ready to be merged. Was that the signal you intended to give?

**Comment by MohamedZaghdoudi** [comment]
hey @MikeMcC399  and  @GisoSchroederSAP,  thanks for the feedback.  
I have already corrected it and uploaded again .

**Review by larswmh — APPROVED** [review]

**Comment by MikeMcC399** [comment]
Please remove or replace the screen-shots in the original post as they are outdated.

File: src/data/faq.json [hunk 8]
```diff
@@ -996,7 +996,7 @@
...
+                                    "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my <a href='#glossary_booster_vaccination' target='_blank'>booster vaccination</a> be accounted for?</a></li><li><a href='#admission_policy_2gplus_not_shown' target='_blank'>Why is the 2G+ status proof not displayed even though I have a vaccination certificate or a certificate of recovery and a negative test?</a></li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:999]: This is an illegal nested link. It should be restored to the original without the nested glossary link.

File: src/data/faq.json [hunk 9]
```diff
@@ -1006,7 +1006,7 @@
...
-                                    "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my booster vaccination be accounted for?</a></li><li><a href='#admission_policy_2gplus_howto' target='_blank'>How do I use the combined 2G+ display?</a></li></ul>"
+                                    "See also: <ul><li><a href='#admission_policy' target='_blank'>General information about the status proof display</a></li><li><a href='#admission_policy_2gplus' target='_blank'>What does the 2G+ status proof mean in concrete terms?</a></li><li><a href='#admission_policy_2gplus_booster' target='_blank'>Will my <a href='#glossary_booster_vaccination' target='_blank'>booster vaccination</a> be accounted for?</a></li><li><a href='#admission_policy_2gplus_howto' target='_blank'>How do I use the combined 2G+ display?</a></li></ul>"
```
Comment (MikeMcC399) [inline src/data/faq.json:1009]: Illegal nested link!

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There are still illegal nested links in this PR. They need to be removed.

**Comment by MikeMcC399** [comment]
Please mark resolved conversations as resolved, so it is clearer which items still need to be addressed.

File: src/data/faq_de.json (hunk 7 above)
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

File: src/data/faq.json (hunk 8 above)
Comment (larswmh) [reply src/data/faq.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

File: src/data/faq.json (hunk 9 above)
Comment (larswmh) [reply src/data/faq.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/167908cf4a3c35b3fad2d6bd099d7f90418dc868, thanks!

**Comment by larswmh** [comment]
Thanks @MikeMcC399 for your extensive review. I believe my latest commit should've sorted out all remaining issue on this PR. Conversations have been closed as suggested.

File: src/data/faq.json (hunk 5 above)
Comment (MikeMcC399) [reply src/data/faq.json]: This one is resolved.

File: src/data/faq_de.json [hunk 10]
```diff
@@ -890,27 +890,27 @@
...
+                                    "Seit Version 2.9 unterstützt die Corona-Warn-App auch den Import von Zertifikaten, die eine <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> dokumentieren. (siehe Blog Artikel: <a href='../../blog/2021-09-08-cwa-version-2-9/' target='_blank' >Corona-Warn-App Version 2.9: Veranstalter*innen können Gäste im Auftrag des Gesundheitsamtes über die Check-in-Funktion warnen</a>)",
+                                    "Seit Version 2.10 erhalten Anwender einen Hinweis, ggf. eine <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> in Betracht zu ziehen. Der Hinweis basiert auf nationalen Regeln, die in Absprache mit dem Robert Koch-Institut (RKI) implementiert wurden. Diese Regeln werden fortlaufend geprüft und entsprechend den Entwicklungen und neuen Erkenntnissen angepasst. (siehe Blog Artikel: <a href='../../blog/2021-09-22-cwa-version-2-10/' target='_blank' >Mit Version 2.10 ist die Corona-Warn-App bereit, schnell über <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfungen</a> zu informieren.</a>)",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:912]: Another illegal nested link here.

**Comment by MikeMcC399** [comment]
@larswmh 

It's very difficult to check the HTML by hand and spot all the issues. In the end I used https://validator.w3.org/ and looked for the error message:

`Start tag a seen but an element of the same type was already open.`

File: src/data/faq_de.json (hunk 10 above)
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/8730e3fa06233ad08580e93253c6a7e9f2f6fb6a, thanks!

File: src/data/faq_de.json [hunk 11]
```diff
@@ -890,27 +890,27 @@
...
...men, und Geburtsdatum) passend zu den bereits existierenden Impfzertifikaten (und auch ggfs. Genesenenzertifikaten) ausgewiesen werden. Nur so kann die Corona-Warn-App das neue Zertifikat zu den anderen Zertifikaten (Impfungen bzw. Genesung) zuordnen (Beispiel: gleicher akademischer Titel oder Angabe des vollständigen Doppelnamens). Alle Zertifikate einer Person sollten in der Corona-Warn-App gruppiert angezeigt werden, siehe Abbildung.</p><p><a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfungen</a> bieten einen sofortigen Impfschutz.</p><p>Basierend auf den bisherigen Impfvorgängen, wird das erste Zertifikat einer <a href='#glossary_auffrischimpfung' target='_blank'>Auffrischimpfung</a> mit '4 von 4' (bei vorheriger <a href='#glossary_basic_immunization' target='_blank'>Grundimmunisierung</a> mit 3 Dosen) ausgestellt.<p><p>Die Kennzeichnung '2 von 1' im Zertifikat einer Impfung zeigt an, dass es sich um den ersten Impfvorgang nach einer Genesung handelt.</p><p>Siehe FAQ-Artikel: <a href='#vac_booster_jj' target='_blank'>Warum erscheint in der Corona-Warn-App eine 14-tägige Wartezeit bis zum vollen Impfschutz nach einer Auffrischimpfung?</a></p></div></div></div>"
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3179/commits/8730e3fa06233ad08580e93253c6a7e9f2f6fb6a, thanks!

**Comment by larswmh** [comment]
@MikeMcC399 

That is a very useful tool, thanks for sharing. After checking with the changes made in the latest commit, I could not find any more issues regarding nested links. Can you confirm?

**Review by MikeMcC399 — APPROVED** [review]
@larswmh 

> After checking with the changes made in the latest commit, I could not find any more issues regarding nested links. Can you confirm?

- I can't see any more link issues from this PR, so I suggest to go ahead and merge.

- I did notice a link issue using the same tool, so I submitted a separate PR https://github.com/corona-warn-app/cwa-website/pull/3207 to fix that one

**Pull request merged**